components/wysiwyg/widget-block-edition/component.js

```diff
@@ -13,16 +13,28 @@ import WidgetList from 'components/widgets/list/WidgetList';
 
 class WidgetBlockEdition extends PureComponent {
   static propTypes = {
-    data: PropTypes.object.isRequired,
     onChangeTab: PropTypes.func.isRequired,
     onChangePage: PropTypes.func.isRequired,
     onChangeSearch: PropTypes.func.isRequired,
-    onSelectWidget: PropTypes.func.isRequired
-  }
+    onSelectWidget: PropTypes.func.isRequired,
+    tab: PropTypes.string.isRequired,
+    loading: PropTypes.bool.isRequired,
+    total: PropTypes.number.isRequired,
+    page: PropTypes.number.isRequired,
```

Comment:
Just my opinion, but I think it makes more sense having all these pagination-related attributes under a `pagination` key. It makes less noise and everything is more organized inside the key. But as I told you before, just my opinion, feel free to change it or not.